resources/sdk/clisdkclient/extensions/cmd/notifications/subscriptions/deletesubscriptions.go

```diff
@@ -0,0 +1,28 @@
+package subscriptions
+
+import (
+	"fmt"
+	"log"
+	"gc/utils"
+	"gc/services"
+
+	"github.com/spf13/cobra"
+)
+
+var deleteSubscriptionsCmd = &cobra.Command{
+	Use:   "delete [channel id]",
+	Short: "Deletes a subscription by Genesys Cloud channel id",
+	Long:  `Deletes a subscription by skill by Genesys Cloud id`,
```

Comment:
The long description references skills. Bad copy/paste?

Reply:
This was a bad copy paste